Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockObjectFactory.cs

```diff
@@ -33,7 +33,7 @@ internal class MockObjectFactory : IPreprocessorObjectFactory
         public TestLogger Logger { get; } = new();
         public MockSonarWebServer Server { get; }
         public Mock<ITargetsInstaller> TargetsInstaller { get; } = new();
-        public MockRoslynAnalyzerProvider AnalyzerProvider { get; } = new() { SettingsToReturn = new AnalyzerSettings { RulesetPath = "c:\\xxx.ruleset" } };
+        public MockRoslynAnalyzerProvider AnalyzerProvider { get; private set; } = new() { SettingsToReturn = new AnalyzerSettings { RulesetPath = "c:\\xxx.ruleset" } };
```

Comment:
pedantic: the private setter is not needed.

```suggestion
        public MockRoslynAnalyzerProvider AnalyzerProvider { get; } = new() { SettingsToReturn = new AnalyzerSettings { RulesetPath = "c:\\xxx.ruleset" } };
```

Reply:
oh yes, I forgot to remove that along with the other code.